artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java

```diff
@@ -241,6 +241,8 @@
 
    private volatile Object providerAgnosticSslContext;
 
+   private int ephemeralPort = 0;
```

Comment:
This should probably be volatile for safety since it seems it will have to be accessed by a different thread in actual use (most likely straight after setting), so unless there is an explicit barrier between them somewhere it would be unreliable otherwise.